cmd/crictl/image.go

```diff
@@ -320,10 +317,25 @@ var imageStatusCommand = &cli.Command{
 		}
 		tmplStr := c.String("template")
 
-		statuses := []statusData{}
-		for i := range c.NArg() {
-			id := c.Args().Get(i)
+		ids := c.Args().Slice()
+
+		if len(ids) == 0 {
+			r, err := ListImages(imageClient, c.String("name"), c.StringSlice("filter"))
+			if err != nil {
+				return fmt.Errorf("listing images: %w", err)
+			}
+			for _, img := range r.GetImages() {
+				ids = append(ids, img.GetId())
```

Comment:
do we need to deduplicate here?

Reply:
I'd say no, because right now it behaves in the same way to output two images when doing something like:

```
sudo crictl inspecti 324bc02ae1231 324bc02ae1231
```